emcc.py

```diff
@@ -1260,6 +1260,9 @@ def check(input_file):
     if shared.Settings.RELOCATABLE:
       shared.Settings.ALLOW_TABLE_GROWTH = 1
 
+    if shared.Settings.WASM2C or shared.Settings.ASYNCIFY or shared.Settings.EMBIND:
+      shared.Settings.USE_LEGACY_DYNCALLS = 1
```

Comment:
why asyncify and why embind? is there a TODO with details somewhere? (or is this very close on your roadmap?)

Reply:
Yup, I will add comments and file bugs.  The plan is to remove this ASAP.

I'm hoping you can help with the `WASM2C` and `ASYNCIFY` work.

Reply:
Sure, I can help with those.